x/voting/keeper/create_vote_test.go

```diff
@@ -29,9 +30,9 @@ func (suite *VotingKeeperTestSuite) TestCreateVote_Cast_Positive_Valid() {
 	app.AccountKeeper.SetAccount(suite.Ctx, acc1)
 	suite.Require().NoError(testutil.FundAccount(app.BankKeeper, suite.Ctx, addr1, balances))
 
-	addr2 := sdk.AccAddress("addr2_______________")
-	acc2 := app.AccountKeeper.NewAccountWithAddress(suite.Ctx, addr2)
-	app.AccountKeeper.SetAccount(suite.Ctx, acc2)
+	validator := app.StakingKeeper.GetAllValidators(suite.Ctx)[0]
```

Comment:
Let's please add invalid test cases as well, and accomodate case where receiver is non-validator which should fail

Reply:
Done